lms/static/sass/base/_reset.scss

```diff
@@ -96,4 +96,5 @@ td { vertical-align: top; }
   @page { margin: 0.5cm; }
   p, h2, h3 { orphans: 3; widows: 3; }
   h2, h3 { page-break-after: avoid; }
+  .donotprint { display:none !important; }
```

Comment:
Indeed, I'm not using .donotprint here. That said, it's one line with zero side effects, it's function is obvious, and it will make implementation a whole lot easier for anybody who wants to use it. Implementing it yourself inside a course is really somewhat annoying.

Would you like me to go through the rest of the code and change all the display:none's to extend such a placeholder selector?